src/features/projectsV2/ProjectDetails/index.tsx

```diff
@@ -143,6 +143,24 @@ const ProjectDetails = ({
     selectedSamplePlantLocation,
   ]);
 
+  const singlePlantInfoProps = {
+    plantData,
+    setSelectedSamplePlantLocation,
+  };
+
+  const plantLocationInfoProps = {
+    plantLocationInfo: hoveredPlantLocation
+      ? hoveredPlantLocation
+      : selectedPlantLocation,
+    setSelectedSamplePlantLocation,
+    isMobile,
+  };
+
+  const projectInfoProps = {
+    project: singleProject,
+    isMobile,
+    setSelectedMode,
+  };
```

Comment:
This wasn't needed IMO, the prop list isn't too lengthy for any of the components, so this actually makes the code less readable.